packages/block-editor/src/hooks/media.scss

```diff
@@ -0,0 +1,61 @@
+.block-editor-hooks__media__inspector-upload-container {
+	position: relative;
+	// Since there is no option to skip rendering the drag'n'drop icon in drop
+	// zone, we hide it for now.
+	.components-drop-zone__content-icon {
+		display: none;
+	}
+}
+
+.block-editor-hooks__media__inspector-upload-container,
+.block-editor-hooks__media__inspector-media-replace-container {
+	button.components-button {
```

Comment:
I feel like we should make this a variation of the button component rather than having a custom rule for it here. That should be a different PR though.